.changeset/itchy-mirrors-hide.md

```diff
@@ -0,0 +1,5 @@
+---
+"@ultraviolet/ui": patch
+---
+
+Autofocus problem on `<Modal />`: found a workaround (see storybook)
```

Comment:
As this is only a story and there is no functional changes, you don't need to push a changeset file 😄